pkg/addons/adminconsole/static/values.tpl.yaml

```diff
@@ -0,0 +1,19 @@
+images:
+    kotsadm: 'proxy.replicated.com/anonymous/kotsadm/kotsadm:{{ index .Images "kotsadm/kotsadm" }}'
```

Comment:
I think you missed my point here but I realize what I said was very unclear. Right now there is a hardcoded list of images in this values file. But your new code discovers images in helm charts. What if a new image gets added? I thought it may be cool to dynamically generate the helm values.yaml overriding images so we would automatically proxy new images. That way we would never make a release with an image we didn't know about and therefore break our rules around whitelisted domains. It was just a thought and I'm happy to approve this pull request as is.

Reply:
Oh I see why this does not make sense now. Since you are parsing the rendered yaml and not the values files. But I do wonder if we could compare the list of discovered images with the list of expect images and somehow warn.

Reply:
oh i see this is not possible since you are extracting images from the rendered yaml but not the values files. as a follow up it may be nice to provide a sanity check, since we have a hardcoded list of images we are expecting, and if we find are any images we do not know about in the rendered template produce an error.